go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/xuri/excelize/v2
 go 1.23.0
 
 require (
+	github.com/mohae/deepcopy v0.0.0-20170929034955-c48cc78d4826
```

Comment:
Cross-referencing my comment in https://github.com/qax-os/excelize/pull/2030#issuecomment-2827510389.

The best case scenario is https://github.com/tiendc/go-deepcopy/issues/34 gets accepted and support for `sync.Map` is added to `github.com/tiendc/go-deepcopy`.